addon/components/new-osf-navbar/template.hbs

```diff
@@ -46,7 +46,20 @@
                             </li>
                         {{else}}
                             {{!SERVICE LINK}}
-                            <li><a href="{{navLink.href}}"> {{navLink.name}}</a></li>
+                            <li>
+                                <a href="{{navLink.href}}">
+                                    {{#if (eq navLink.type 'addAPreprint')}}
+                                        <span class="hidden-xs hidden-sm">
+                                            {{navLink.name}}
+                                        </span>
+                                        <span class="hidden-md hidden-lg hidden-xl">
```

Comment:
Seems like this could be simplified using `hidden-md-up`. The other case (4 lines above) could be simplified using `hidden-sm-down`: https://v4-alpha.getbootstrap.com/layout/responsive-utilities/.
Unless that is not available in our current version of bootstrap, which I suspect is the case :(